spec/unit/rom/plugin_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe "ROM::PluginRegistry" do
+
+  subject(:env) { setup.finalize }
+
+  let(:setup) { ROM.setup(:memory) }
+
+  before do
+    GlobalExtension = Module.new
```

Comment:
You can wrap it in `Test::GlobalExtension` so that our rspec callbacks will clean it from the runtime